test/TestConfig.py

```diff
@@ -15,28 +15,135 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import unittest
+from unittest import skip
 
 from test.TopydoTestCase import TopydoTest
 from topydo.lib.Config import config
 
 
 class ConfigTest(TopydoTest):
-    def test_config1(self):
-        self.assertEqual(config("test/data/config1").default_command(), 'do')
+    def test_config01(self):
+        self.assertEqual(config("test/data/ConfigTest1.conf").default_command(), 'do')
 
-    def test_config2(self):
+    def test_config02(self):
         self.assertNotEqual(config("").default_command(), 'do')
 
-    def test_config3(self):
-        self.assertTrue(config("test/data/config2").ignore_weekends())
+    def test_config03(self):
+        self.assertTrue(config("test/data/ConfigTest2.conf").ignore_weekends())
 
-    def test_config4(self):
+    def test_config04(self):
         """ Test that value in file is overridden by parameter. """
         overrides = {
             ('topydo', 'default_command'): 'edit'
         }
 
-        self.assertEqual(config("test/data/config1", p_overrides=overrides).default_command(), 'edit')
+        self.assertEqual(config("test/data/ConfigTest1.conf",
+                                p_overrides=overrides).default_command(),
+                         'edit')
+
+    def test_config05(self):
+        """ Bad colour switch value. """
+        # boolean settings must first be typecast to integers, because all
+        #  strings evaulate to 'True'
+        self.assertEqual(config("test/data/ConfigTest4.conf").colors(),
+                         bool(int(config().defaults["colors"])))
```

Comment:
I think these tests need to be fixed after the inclusion of PR #58 . Better is perhaps to have a dedicated PR for this as well, since it's not strictly related to the `top` command.